pkgs/top-level/all-packages.nix

```diff
@@ -5840,7 +5840,7 @@ with pkgs;
 
   monetdb = callPackage ../servers/sql/monetdb { };
 
-  monado = callPackage ../applications/graphics/monado {
+  monado = callPackage ../by-name/mo/monado/package.nix {
     inherit (gst_all_1) gstreamer gst-plugins-base;
   };
```

Comment:
```suggestion
```

Packages in `by-name` are automatically called.

Reply:
This is here as the package function expects `gstreamer` `gst-plugins-base` as package arguments.

Reply:
You can use `gst_all_1` as the argument name.